compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/EnumClassLowering.kt

```diff
@@ -162,6 +163,13 @@ private class EnumClassLowering(val context: JvmBackendContext) : ClassLoweringP
                     IrGetValueImpl(expression.startOffset, expression.endOffset, it.type, it.symbol, expression.origin)
                 } ?: expression
 
+            override fun visitSetValue(expression: IrSetValue): IrExpression {
```

Comment:
Real change: when remapping parameter symbols we now need to look at SetValue as well as GetValue.